src/main.mjs

```diff
@@ -9,10 +9,21 @@ const syntaxMapping = {
   scss: postcssScss,
 };
 
+function resolveSorterOption({
+  cssDeclarationSorterCustomOrder,
+  cssDeclarationSorterOrder,
+}) {
+  return cssDeclarationSorterCustomOrder.length > 0
```

Comment:
Seems like Prettier's option resolve always makes this an array and errors out if a different type is passed so all we have to check is the length to see if it's in use I think.